app/policies/project_policy.rb

```diff
@@ -117,6 +117,22 @@ def transfer_sample_into_project?
     false
   end
 
+  def clone_sample?
+    return true if Member.user_has_namespace_maintainer_access?(user, record.namespace, false)
```

Comment:
This line isn't required as a maintainer and owner can both clone samples. The `can_clone_sample?` method can be updated to match `can_clone_sample_to_project?` in `member.rb`

Reply:
26d436d35e40ad191ed3b6a465ff7f3e9412c4f6